curriculum/js1/8.js

```diff
@@ -5,8 +5,18 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, c=a) => {
+  if (c<=1) {
+    return c;
+  }
+  if ((a%c)===0) {
+    if ((b%c)===0) {
+      return c
+    }
+    return solution(a,b,c-1)
+  }
+  return solution(a,b,c-1)
```

Comment:
* Please add a semicolon to the end of this statement.
* Adding spaces after commas will improve the readability of this code

Reply:
Could you check the changes? Thanks!